appendices/assignments.md

```diff
@@ -0,0 +1,211 @@
+is to modify this example to focus on weather from a local city, 
+a specific time period, or different aspects of the weather (e.g. temperature only).
+- While the project package does not necessarily need to involve data, it might
+be more motivating for the learners to work on if the project does include data. 
+R is a language predominantly used by researchers and data scientists
+for doing data analysis, visualization, and related reporting. Given this,
+learners taking this course are probably already familiar with these tasks and
+would benefit more from the course if they see and use an application they are
+familiar with.
+
+TODO: Add some examples that instructors can based off of and deviate if needed.
+
+Here are some example datasets that could form the basis for the project packages:
+
+- [NYC Dog Licensing dataset](https://data.cityofnewyork.us/Health/NYC-Dog-Licensing-Dataset/nu7n-tubp)
+- [Statistics Canada](https://www150.statcan.gc.ca/n1/en/type/data)
+- The openly licensed [Airbnb data](http://insideairbnb.com/get-the-data.html)
+- [Atmospheric CO~2~](https://scrippsco2.ucsd.edu/data/atmospheric_co2/index.html)
+
+- [fivethirty eight article about visits to national parks](https://fivethirtyeight.com/features/the-national-parks-have-never-been-more-popular/).  A package might include the data (which is a bit painful to get from the NPS), plus a 
+
+- (For advanced users) [Denmark Statistics](https://www.statbank.dk/statbank5a/default.asp?w=1920) 
+(through the R package [dkstat](https://github.com/rOpenGov/dkstat) or the 
+[StatBank API](https://www.dst.dk/en/Statistik/brug-statistikken/muligheder-i-statistikbanken/api))
+
+Other potential examples to use for non-dataset-based project packages include:
+
+- A simple package for working with missing values. Functions within it could
+include simpler versions of other functions like `naniar::count_missing()`,
+`tidyr::replace_na()`, or `dplyr::na_if()`. The data to work with could be
+anything that has missing values (or should have missing values), primarily to use
+as a way to build and demonstrate the functions.
+
+- Simple simulations like cellular automata in [Nature of Code](https://natureofcode.com/). The
+package might have a function for plotting a vector of zeros and one, functions
+for evolving such a vector, and functions that the control evolution.
+
+- (Advanced users) A package for "branded" analysis, for instance where the data
+might be corporate/team/school colors and functions that might include a custom
+ggplot2 theme for them.
+
+### Second assignment
+
+The learners should either be given a list of some potential packages to improve
+or be given detailed instructions on how to find potential packages. If you want
+to provide a list of possible packages, here is some instructions on how to find
+these packages to give the learners:
+
+TODO: Include instructions here, eg what search terms to use in GitHub etc and how to find packages.
```

Comment:
Would also add CRAN Views as a place to search for packages given that it categorizes them by domain

Reply:
Done.